src/resources/i18n/en.json

```diff
@@ -45,7 +45,14 @@
     "disease1": "Diagnostic 1",
     "disease2": "Diagnostic 2",
     "disease3": "Diagnostic 3",
-    "validatedisease": "This diagnostic should be different"
+    "validatedisease": "This diagnostic should be different",
+    "code": "Code",
+    "updateopd": "update opd",
+    "saveopd": "save opd",
+    "created": "Opd Created",
+    "updated": "Opd updated",
+    "createsuccess": "Opd has been created successfully!",
+    "updatesuccess": "Opd {{code}} has been updated successfully!"
```

Comment:
Here I wouldn't use the code, is there something else of more useful inside the object?

Reply:
May be the date? But the date is not unique. May be the diseases names? It could take more place than expected. May be the complain? It could take more place and can also be empty. That's why I choose the code property.

Reply:
Ok we definitely keep the code. If we didn't do yet so in this case we should put the code inside the OPD Table in order to avoid some users delete mistake.

Reply:
I already put the code in the table.